promptsource/templates/tydiqa/primary_task/templates.yaml

```diff
@@ -1,127 +1,174 @@
-      original_task: true
+      choices_in_prompt: false
+      metrics:
+      - Accuracy
+      original_task: false
     name: yes_no_question
     reference: Ask if this is a yes no question
-  6c2e002a-a582-44cc-971e-3919585072d7: !Template
+  7b8b7707-dbad-40d2-a5c2-430e6ace10bb: !Template
     answer_choices: null
-    answer_choices_key: null
-    id: 6c2e002a-a582-44cc-971e-3919585072d7
-    jinja: "{% set _position = [\"above\", \"following\"] |random %} \n{% if language\
-      \ == \"english\" %} \n    {% if annotations.yes_no_answer[0] == \"YES\" or annotations.yes_no_answer[0]\
-      \ == \"NO\" %} \n        {% if  _position == \"above\" %} \n            Question:\
-      \ {{question_text}} {{\"\\n\"}} \n        {% endif %} \n        Answer the question\
-      \ from Wikipedia. \n        {% if  _position == \"following\" %} \n        \
-      \    {{\"\\n\"}} Question: {{question_text}} \n        {% endif %} \n||| \n\
-      {{annotations.yes_no_answer[0]}} \n    {% endif %} \n{% endif %}"
+    answer_choices_key: Yes ||| No ||| None
+    id: 7b8b7707-dbad-40d2-a5c2-430e6ace10bb
+    jinja: '{% if language == "english" %}
+
+      Answer the following question with "{{answer_choices[0]}}" or "{{answer_choices[1]}}"
+      or "{{answer_choices[2]}}" if none of the first two answers apply.
+
+      Question: {{question_text}}
+
+      Topic: {{document_title}}
+
+      Article: {{document_plaintext}}
+
+      |||
+
+      {{annotations.yes_no_answer[0] | capitalize}}
+
+      {% endif %}'
     metadata: !TemplateMetadata
-      _do_eval: false
-      _do_train: false
-      choices_in_prompt: null
-      metrics: []
+      choices_in_prompt: true
+      metrics:
+      - Accuracy
       original_task: true
-    name: binary_question_without_context_wikipedia
-    reference: Ask a yes/no question mentioning KB.
+    name: read_and_answer
+    reference: ''
   9c42e3fd-d46e-4149-bb60-4b3118104d95: !Template
     answer_choices: null
-    answer_choices_key: null
+    answer_choices_key: Yes ||| No
     id: 9c42e3fd-d46e-4149-bb60-4b3118104d95
     jinja: "{% if language == \"english\" %} \n    {% if annotations.yes_no_answer[0]\
-      \ == \"YES\" or annotations.yes_no_answer[0] == \"NO\" %} \n        After reading\
-      \ the following text snippet from Wikipedia, please answer the question: {{question_text}}\
-      \ \n        {{\"\\n\"}}{{document_plaintext}}\n||| \n{{annotations.yes_no_answer[0]}}\n\
+      \ == \"YES\" or annotations.yes_no_answer[0] == \"NO\" %} \nAfter reading the\
+      \ following text snippet from Wikipedia, please answer the question: {{question_text}}\
+      \ \n{{document_plaintext}}\n||| \n{{annotations.yes_no_answer[0] | capitalize}}\n\
       \    {% endif %}\n{% endif %}"
     metadata: !TemplateMetadata
       _do_eval: false
       _do_train: false
-      choices_in_prompt: null
-      metrics: []
+      choices_in_prompt: false
+      metrics:
+      - Accuracy
       original_task: true
```

Comment:
The prompt `open_domain_qa_without_choices` has `original_task: false`, but it seems it is no less original-tasky than this one. Should one or the other be changed?